HACKING.rst

```diff
@@ -1,44 +1,52 @@
-The first time you start working in the repository, set up a clean test
-environment (Before you start make sure that you have setup a database
-to be used by for HaaS. HaaS supports SQLIte and PostgreSQL databases.
-You should setup either of the two before you start further. Details to
-setup a database can be found in INSTALL.rst)::
+Setup Test Environment
+++++++++++++++++++++++
 
-  virtualenv .venv
+Install Prerequisite Software
+=============================
 
-Next, each time you start working, enter the environment::
+HaaS requires a number of packages available through the CentOS RPM
+repositories, as well as the EPEL repository. EPEL can be enabled via::
 
-  source .venv/bin/activate
+    yum install epel-release
 
-Then, proceed with installing the HaaS and its dependencies into the virtual
-environment::
+Then, the rest of the packages can be installed via::
 
-  pip install -e .
+    yum install libvirt bridge-utils ipmitool telnet httpd mod_wsgi python-pip qemu-kvm python-virtinst virt-install python-psycopg2 vconfig net-tools
 
-On systems with older versions of ``pip``, such as Debian Wheezy and Ubuntu
-12.04, this install will fail with the following error::
+In addition, HaaS depends on a number of python libraries. Many of these are
+available as RPMs as well, but we recommend installing them with pip, since
+this will install the versions that HaaS has been tested with.  This is done
+automatically by the instructions below.
 
-  AttributeError: 'NoneType' object has no attribute 'skip_requirements_regex'
+Creating HaaS System User
+=========================
 
-Fix this by upgrading ``pip`` within the virtual environment::
+Create a system user ``haas_user`` with::
```

Comment:
I always wanted the user to be simply `haas` and not `haas_user`.
If you have no objection against it, I suggest changing it here.